kotest-assertions/kotest-assertions-livedata/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+   kotlin("multiplatform")
+   id("com.android.library")
+   id("com.adarshr.test-logger")
+}
+
+repositories {
+   jcenter()
+}
+
+android {
+   compileSdkVersion(30)
+   defaultConfig {
+      minSdkVersion(19)
+      targetSdkVersion(30)
+   }
+
+   compileOptions {
+      sourceCompatibility = JavaVersion.VERSION_1_8
+      targetCompatibility = JavaVersion.VERSION_1_8
+   }
+
+   testOptions {
+      unitTests.isReturnDefaultValues = true
+   }
+}
+
+kotlin {
+   android {
+      compilations.all {
+         kotlinOptions {
+            jvmTarget = "1.8"
+         }
+      }
+   }
+   sourceSets {
+      targets.all {
+         compilations.all {
+            kotlinOptions {
+               freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
+            }
+         }
+      }
+
+      val androidMain by getting {
+         dependencies {
+            implementation(project(Projects.AssertionsShared))
+            implementation(project(Projects.AssertionsApi))
+            implementation(project(Projects.Engine))
+            implementation(project(Projects.extension("livedata")))
+            implementation(AndroidLibs.AndroidX.coreKtx)
+            implementation(AndroidLibs.AndroidX.Lifecycle.runtimeKtx)
+            implementation(AndroidLibs.AndroidX.Lifecycle.liveDataKtx)
+            implementation(AndroidLibs.AndroidX.Lifecycle.viewModelKtx)
+            implementation(AndroidLibs.AndroidX.ArchCore.testing)
+         }
+      }
+
+      val androidTest by getting {
+         dependencies {
+            implementation(project(Projects.JunitRunner))
+         }
+      }
+   }
+}
+
+tasks.withType<Test> {
+   useJUnitPlatform()
+   filter {
+      isFailOnNoMatchingTests = false
+   }
+   testLogging {
+      showExceptions = true
+      showStandardStreams = true
+      events = setOf(
+         TestLogEvent.FAILED,
+         TestLogEvent.PASSED
+      )
+      exceptionFormat = TestExceptionFormat.FULL
+   }
+}
+
+//apply(from = "../../publish-mpp.gradle.kts")
```

Comment:
I need to edit this file since it doesn't support android libraries for now (to add an if to it to add conditionally the java/java-library plugins since they're not supported by the `com.android.library` library one.